code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -38,19 +38,27 @@ namespace WijmoProvider.Feature {
             s: wijmo.grid.FlexGrid,
             e: wijmo.grid.CellRangeEventArgs
         ): void {
-            const binding = s.getColumn(e.col).binding;
+            const column = s.getColumn(e.col);
+            const OSColumn = this._grid
+                .getColumns()
+                .find((item) => item.provider.index === column.index);
+
             const newValue = s.getCellData(
                 e.row,
                 e.col,
-                this._grid.getColumn(binding).columnType ===
-                    OSFramework.Enum.ColumnType.Dropdown
+                OSColumn.columnType === OSFramework.Enum.ColumnType.Dropdown
             );
             // The old value can be captured on the dirtyMark feature as it is the one responsible for saving the original values
             const oldValue = this._grid.features.dirtyMark.getOldValue(
                 e.row,
-                binding
+                column.binding
+            );
+            this._triggerEventsFromColumn(
+                e.row,
+                column.index,
```

Comment:
We should be passing column uniqueId instead of column index.

Reply:
@JoaoFerreira-FrontEnd This is still missing, right?